doc/getting_started/installation_mac.rst

```diff
@@ -163,10 +163,11 @@ When mounted, the file system of the image will be available under
 Setting the Toolchain Options
 =============================
 
-In the Zephyr kernel source tree we provide two configurations for
-both ARM and X86 that can be used to preselect the options needed
-for building the toolchain.
-The configuration files can be found in :file:`${ZEPHYR_BASE}/scripts/cross_compiler/`.
+In the Zephyr kernel source tree we provide configurations for NIOS-II and
+X86 that can be used to preselect the options needed for building the toolchain.
+
```

Comment:
Should you add Anas' comment here:

     For ARM we recommend using https://developer.arm.com/open-source/gnu-toolchain/gnu-rm/downloads.

Reply:
it is already mentioned in the same page in another section. I do not really care, but this will be duplication and another reference to worry about.

Reply:
sorry... I was just looking at the change.  Don't need to duplicate.